samples/drivers/lora/send/src/main.c

```diff
@@ -25,13 +24,12 @@ char data[MAX_DATA_LEN] = {'h', 'e', 'l', 'l', 'o', 'w', 'o', 'r', 'l', 'd'};
 
 void main(void)
 {
-	const struct device *lora_dev;
+	const struct device *lora_dev = DEVICE_DT_GET(DEFAULT_RADIO_NODE);
 	struct lora_modem_config config;
 	int ret;
 
-	lora_dev = device_get_binding(DEFAULT_RADIO);
-	if (!lora_dev) {
-		LOG_ERR("%s Device not found", DEFAULT_RADIO);
+	if (!device_is_ready(lora_dev)) {
+		LOG_ERR("%s Device not found", lora_dev->name);
```

Comment:
We could change the message to "%s Device not ready", here and in a few more places in this PR.